cli/install/install.go

```diff
@@ -434,20 +471,42 @@ func installTt(binDir string, installCtx InstallCtx, distfiles string) error {
 		}
 
 		if !versionFound {
-			return fmt.Errorf("%s version of tt doesn't exist", ttVersion)
+			commits, err_hashes := getTTCommits(installCtx.Local, distfiles)
```

Comment:
Do not mix camel and snake case names styles.

Reply:
fixed